### PR TITLE
7.2-Preview to GA

### PR DESCRIPTION
**Pull request by jlichwa**
Copy 7.2-Preview to Stable and remove '-preview' suffix from all files

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  -  [ ] Ensure to copy the existing version into new directory structure for first commit (including refactoring) and then push new changes including version updates in separate commits. This is required to review the changes efficiently.
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/keyvault/data-plane/readme.md [hunk 1]
```diff
@@ -45,6 +45,23 @@ input-file:
...
   - Microsoft.KeyVault/preview/7.3-preview/storage.json
 ```
+
+### Tag: package-7.2
+
+These settings apply only when `--tag=package-7.2` is specified on the command line.
+
+``` yaml $(tag) == 'package-7.2'
+input-file:
+- Microsoft.KeyVault/preview/7.2/certificates.json
```
Comment (heaths) [inline specification/keyvault/data-plane/readme.md:55]: These should reference the "stable" folder.

Comment (heaths) [reply specification/keyvault/data-plane/readme.md:55]: Once thatis done, I should be able to use the OpenAPI Hub Diff'ing tool to make sure things are correct. I suspect you just copied the files and changed the api-version, but want to verify everything to sign off, of course.

**Review by heaths — CHANGES_REQUESTED** [review]

File: specification/keyvault/data-plane/readme.md (hunk 1 above)
Comment (jlichwa) [reply specification/keyvault/data-plane/readme.md:55]: Thanks

**Comment by heaths** [comment]
Based on https://portal.azure-devex-tools.com/tools/diff/eyJzb3VyY2VVcmwiOiJodHRwczovL2dpdGh1Yi5jb20vQXp1cmUvYXp1cmUtcmVzdC1hcGktc3BlY3MvYmxvYi9tYXN0ZXIvc3BlY2lmaWNhdGlvbi9rZXl2YXVsdC9kYXRhLXBsYW5lL3JlYWRtZS5tZCIsInRhcmdldFVybCI6Imh0dHBzOi8vZ2l0aHViLmNvbS9BenVyZS9henVyZS1yZXN0LWFwaS1zcGVjcy9ibG9iL2NkNDc0OTRmM2I3ZGFhMmU5MTQxNzhiNTJiYWQ1ZWRlYmUzNWQwODkvc3BlY2lmaWNhdGlvbi9rZXl2YXVsdC9kYXRhLXBsYW5lL3JlYWRtZS5tZCIsInNvdXJjZVRhZyI6InBhY2thZ2UtNy4xIiwidGFyZ2V0VGFnIjoicGFja2FnZS03LjIifQ==, there are some actual breaking changes from 7.1 stable:

1. `exportable` was dropped from `KeyProperties`

Other than that, changes look expected. Be sure to go through the other validation errors reported.

**Comment by heaths** [comment]
@herveyw-msft you originally added `exportable` to `KeyProperties` for SKR, right? Or was that separate? It found it's way into 7.1 stable, and was removed in 7.2-preview (which we're trying to move to stable). It shouldn't hurt keeping, right? Will the service err if passed it, though?

/cc @Lusitanian @vickm

**Comment by herveyw-msft** [comment]
@heaths If we can, we should get rid of exportable in any Swagger version prior to 7.3-preview.

**Comment by jlichwa** [comment]
> @heaths If we can, we should get rid of exportable in any Swagger version prior to 7.3-preview.

@herveyw-msft Is exportable going to be in 7.3 or it will be changed to something else. 
In case of 7.1 all SDKs and tools will have exportable flag and there is no option to reverse that. 
Does service break when exportable is passed today? If not, it could be better to leave it and just document that is ignored by service rather than introducing breaking change between version 7.1 and 7.2 just to add it later anyway in 7.3.... @heaths any thoughts?

**Comment by herveyw-msft** [comment]
@jlichwa , @heaths : exportable is part of 7.3-preview. I cannot speak to AKV behaviour if this is sent to AKV; for Managed HSM which already implements this part of 7.3-preview, it would be accepted with correct semantics. The real problem is that it should not be part of any SDK/PSH/CLI prior to 7.3-preview.

**Comment by jlichwa** [comment]
> @jlichwa , @heaths : exportable is part of 7.3-preview. I cannot speak to AKV behaviour if this is sent to AKV; for Managed HSM which already implements this part of 7.3-preview, it would be accepted with correct semantics. The real problem is that it should not be part of any SDK/PSH/CLI prior to 7.3-preview.

I completely understand, but somehow it ended up in 7.1, which means it is part of GA SDKs etc... So I believe that is unremovable at this point (it will require long multi year deprecation process). @heaths please correct if I'm wrong.

**Comment by heaths** [comment]
Yes, removing it is a breaking change. That said, it seems (at least in .NET, but we do API unification reviews so likely all languages) it's not in the SDK, so I think it's fine to leave defined. A customer using the swaggers directly could still send it, but it's been there since 7.1. Removing it could be worse.

**Review by heaths — APPROVED** [review]

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**